packages/wonder-blocks-dropdown/components/single-select-menu.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+// A menu that consists of action items
+
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import DropdownCore from "./dropdown-core.js";
+import SelectBox from "./select-box.js";
+import SelectItem from "./select-item.js";
+import SeparatorItem from "./separator-item.js";
+
+import type {SelectItemProps, SeparatorProps} from "../utils/types.js";
+
+type ItemProps = SelectItemProps | SeparatorProps;
+
+type Props = {
+    /**
+     * The items in this menu.
+     */
+    items: Array<ItemProps>,
+
+    /**
+     * Callback for when the selection of the menu changes. Parameter is the
+     * newly selected item.
+     */
+    onChange: (selectedValue: string) => void,
+
+    /**
+     * Placeholder for the opening component when there are no items selected.
+     */
+    placeholder: string,
+
+    /**
+     * Whether this menu should be left-aligned or right-aligned with the
+     * opener component. Defaults to left-aligned.
+     */
+    alignment?: "left" | "right",
+
+    /**
+     * Whether to display the "light" version of this component instead, for
+     * use when the item is used on a dark background.
+     */
+
+    light?: boolean,
+
+    /**
+     * Whether this menu is disabled. A disabled menu may not be opened and
+     * does not support interaction. Defaults to false.
+     */
+    disabled?: boolean,
+
+    /**
+     * Optional styling to add to dropdown.
+     */
+    style?: any,
+};
+
+type State = {
+    /**
+     * Whether or not menu is open.
+     */
+    open: boolean,
+
+    /**
+     * The value and label of the selected element.
+     */
+    selected: {
+        value: string,
+        label: string,
+    },
```

Comment:
Move to example.  See comments in docs.md.  Also, we only really need the `value` since the parent can always look up the `label` if needed.